source/tools/configuration.py

```diff
@@ -43,52 +48,47 @@ def __init__(self):
     @classmethod
     def initialize(cls, config):
         """
-        Initialize general keys for this host
+        Initialize general keys for this ASD Manager
         :param config: The configuration containing API IP:port, ASD IPs and ASD start port
         :type config: dict
-        :return: Node id
+        :return: Node ID
         :rtype: str
         """
-        node_id = ''.join(random.choice(string.ascii_letters + string.digits) for _ in range(32))
-        password = ''.join(random.choice(string.ascii_letters + string.digits) for _ in range(32))
-        base_config = {'config/main': {'node_id': node_id,
-                                       'password': password,
-                                       'username': 'root',
-                                       'ip': config['api_ip'],
-                                       'port': config['api_port'],
-                                       'version': 0},
-                       'config/network': {'ips': config['asd_ips'],
-                                          'port': config['asd_start_port']}}
-        for key, value in base_config.iteritems():
-            cls.set(key='/ovs/alba/asdnodes/{0}/{1}'.format(node_id, key),
-                    value=value,
-                    raw=False)
+        with open('/etc/openvstorage_sdm_id', 'r') as the_file:
+            node_id = the_file.read().strip()
+            cls.set(key=Configuration.ASD_NODE_CONFIG_MAIN_LOCATION.format(node_id),
```

Comment:
No need to have this indented under the open file as the only thing that had to be done was reading the id